src/Build/BackEnd/Components/Logging/EventSourceSink.cs

```diff
@@ -27,92 +25,92 @@ internal sealed class EventSourceSink :
         /// <summary>
         /// This event is raised to log a message.
         /// </summary>
-        public event BuildMessageEventHandler MessageRaised;
+        public event BuildMessageEventHandler? MessageRaised;
```

Comment:
why use the old event versus reactive extensions?

Reply:
IIRC this was discussed earlier on a different PR already. 
* Rx.NET needs additional libs (not used accross VS or sdk) - which would require perf justification
* Introducing new paradigm to existing code base with diferent paradigm is net negative (there would be two paradigms or a very extensive refactoring of the whole codebase would be needed). In a new codebase it can be great
* This would be breaking to a fundamental public API (`IEventSource`)